CHANGELOG.asciidoc

```diff
@@ -28,6 +28,21 @@ Notes:
 [[release-notes-3.x]]
 === Node.js Agent version 3.x
 
+==== Unreleased
+
+[float]
+===== Breaking changes
+
+[float]
+===== Features
+
+* Added <<span-frames-min-duration, `spanFramesMinDuration`>>
+  configuration field, allowing users to set a time threshold value that spans
+  must reach before the agent will add a stack trace to the span.
+
+[float]
```

Comment:
Separately, the "Span Stack Traces" section of performance.asciidoc should be updated to mention this option.  IIUC the state will be:
- captureSpanStackTraces off: Least overhead.
- captureSpanStackTraces=true, spanFramesMinDuration=$somePositiveValue: Still some overhead for each span as `Error.captureStackTrace` is called on span creation.

I think we should get some sanity perf numbers to validate whether the overhead is low enough. That could be on a separate ticket to decide if we'd *still* want `captureSpanStackTraces` *off* by default.

Reply:
_nod_ that makes sense.  I've added some initial words to `performance.asciidoc` about this new configuration value, and have also created an issue to capture the intent around turning this off by default: https://github.com/elastic/apm-agent-nodejs/issues/2092